### DISCUSSION TITLE
Release open source versie van OneGround

### DISCUSSION THREAD

**Discussion by Hugo-ter-Doest** [OneGround/ZGW-APIs#14] [Q&A]
Kunnen jullie al iets zeggen over verwachte publicatiedatum van de open source versie van OneGround?

**Comment by johannesbattjes** [comment]
Niet echt. We moeten de code (met name het configuratiedeel) nog splitsen in een deel dat samenhangt met ons gebruikers/organisatiebeheer en een deel dat werkt zonder die samenhang (en dus op een willekeurige plek kan draaien). We zijn daar nu mee bezig.
Gaat het jou om de code @Hugo-ter-Doest of meer om bijvoorbeeld een werkende docker-image?

**Comment by Hugo-ter-Doest** [comment]
Om beide. De Docker om het werkend te zien, en te kijken of we gemakkelijk kunnen aansluiten met onze componenten. En de code repository om naar de tech stack en documentatie te kijken.

**Comment by johannesbattjes** [comment]
OK goed om te weten. 
Voor dit specifieke doel "te kijken of we gemakkelijk kunnen aansluiten met onze componenten" kunnen we  je ook een account geven in onze preproductieomgeving waarmee je direct kunt testen. Heb je daar interesse in?
